qdax/baselines/sac.py

```diff
@@ -299,21 +352,71 @@ def eval_policy_fn(
             play_step_fn=play_step_fn,
         )
 
-        eval_metrics_key = CompletedEvalWrapper.STATE_INFO_KEY
```

Comment:
@Egiob was there a reason for those lines? @ranzenTom has replaced it by just a line. Is it legit or are there some cases that won't be handled correctly?

Reply:
Yeah basically Brax's EvalWrapper handles the case where multiple episodes are done within an `episode_length` rollout, so that it will average the return of theses episodes. I guess it is not a big problem to just take the return of the first episode (accept the changes proposed by Thomas).

Reply:
ok, thanks